tests/vm_test.go

```diff
@@ -2087,15 +2054,7 @@ func startVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtu
 
 func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.VirtualMachine {
 	By("Stopping the VirtualMachine")
-	err := tests.RetryWithMetadataIfModified(vm.ObjectMeta, func(meta k8smetav1.ObjectMeta) error {
-		vm, err := virtClient.VirtualMachine(meta.Namespace).Get(context.Background(), meta.Name, k8smetav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		vm.Spec.Running = nil
-		runStrategyHalted := v1.RunStrategyHalted
-		vm.Spec.RunStrategy = &runStrategyHalted
-		_, err = virtClient.VirtualMachine(meta.Namespace).Update(context.Background(), vm, metav1.UpdateOptions{})
-		return err
-	})
+	err := patchVmRunStrategy(vm, v1.RunStrategyHalted)
```

Comment:
One liner: `Expect(patchVmRunStrategy(vm, v1.RunStrategyHalted)).To(Succeed())`

Reply:
Done here: https://github.com/kubevirt/kubevirt/pull/11948.